themes/classic/templates/catalog/_partials/product-details.tpl

```diff
@@ -50,7 +50,11 @@
           <dl class="data-sheet">
             {foreach from=$product.features item=feature}
               <dt class="name">{$feature.name}</dt>
-              <dd class="value">{$feature.value}</dd>
+              {if $feature.id_feature === '4'}
```

Comment:
Yes, agree. We will do this with weight product (on shipping), not feature. It makes more sense isn't ?